### PR TITLE
Updates Stocks Intro Guide for Terminal 2.0

### PR DESCRIPTION
**Pull request by deeleeramone**
Updates information, screenshots, and examples to be current for version 2.0

### CODE REVIEW

File: website/content/terminal/guides/intros/stocks/index.md [hunk 1]
```diff
@@ -1,225 +1,364 @@
...
 ```
-2022 May 17, 08:46 (🦋) /stocks/ $ load FB
 
-Loading Daily FB stock with starting period 2019-05-13 for analysis.
+Data can also be exported directly from the `load` function as a CSV, JSON, or XLSX file. The file is created in the OpenBBUserData folder.
 
-Datetime: 2022 May 17 08:49
-Timezone: America/New_York
+```console
+load spy -s 1990-01-01 -m --export spy_monthly.csv
```
[the remark this answers is not in the document]
Comment (JerBouma) [reply website/content/terminal/guides/intros/stocks/index.md:158]: This also doesn't seem to work:

```
2022 Dec 05, 03:59 (🦋) /stocks/ $ load spy -s 1990-01-01 -m --export spy_monthly.csv

usage: load -t TICKER [-s START] [-e END] [-i {1,5,15,30,60}] [-p] [-f FILEPATH] [-m] [-w] [-r {ytd,1y,2y,5y,6m}] [-h] [--export {csv,json,xlsx}] [--source {YahooFinance,IEXCloud,AlphaVantage,Polygon,EODHD}]
load: error: argument --export: invalid choice: 'spy_monthly.csv' (choose from 'csv', 'json', 'xlsx')
```

File: website/content/terminal/guides/intros/stocks/index.md [hunk 2]
```diff
@@ -1,225 +1,364 @@
...
-<img src="https://user-images.githubusercontent.com/46355364/169503960-62a59aa2-1dbd-46f1-9ea0-095817d7da5d.png" alt="Stocks Menu with Ticker" width="800"/>
+```console
+/stocks/load SPY -s 1990-01-01 -m/candle
+```
 
-### Sub-menus available
+![candle](https://user-images.githubusercontent.com/85772166/205148777-cd7379f1-b7ef-41b8-890a-d0cf24e725d3.png "candle")
 
-The Stock menu has an extensive list of available sub-menus ranging from fundamental analysis to sector analysis to advanced prediction techniques. To find more information about each menu, click on one of the following:
+The help dialgoe for the `candle` command shows how this chart can be supplemented with additional data; specifically, these arguments:
```
Comment (JerBouma) [inline website/content/terminal/guides/intros/stocks/index.md:181]: dialogue*

File: website/content/terminal/guides/intros/stocks/index.md [hunk 3]
```diff
@@ -1,225 +1,364 @@
...
 
-### Examples
+Be sure to adjust the values for moving averages to correspond with the interval of the data loaded. Below adds movings averages for one and four year periods (because data loaded is monthly), and changes the y-axis to a log-scale.
 
-If we want to a look at the historical data and fundamentals of Microsoft, we can do the following, starting from the `stocks` menu. First, load in the ticker of Microsoft, this is `MSFT` and can be found with the <a href="/terminal/reference/stocks/search/" target="_blank" rel="noreferrer noopener">search</a> command:
+```console
+candle --ma 12,48 --log
+```
 
+!![candle](https://user-images.githubusercontent.com/85772166/205148707-f1f2172f-8dee-4f31-b020-4f8043ccccd7.png "candle")
```
Comment (JerBouma) [inline website/content/terminal/guides/intros/stocks/index.md:196]: Let's change this image to the original image.

File: website/content/terminal/guides/intros/stocks/index.md [hunk 4]
```diff
@@ -1,225 +1,364 @@
...
-| MSFT.SN   | Microsoft Corporation              | United States | Technology | Software - Infrastructure | Chile          |
-| MSFT.VI   | Microsoft Corporation              | United States | Technology | Software - Infrastructure | Austria        |
-| MSFT34.SA | Microsoft Corporation              | United States | Technology | Software - Infrastructure | Brazil         |
-| SMSF.L    | Leverage Shares -1x Microsoft ETC  | None          | None       | None                      | United-Kingdom |
+### Codes
+
+`codes` prints the CIK, Composite FIGI, Share Class FIGI, and SIC codes - when available - for a US-listed stock. This function requires a free API key from Polygon.
+
+```console
+load aapl/codes
```
Comment (JerBouma) [inline website/content/terminal/guides/intros/stocks/index.md:307]: Let's place this in a separate code block so people can copy.

File: website/content/terminal/guides/intros/stocks/index.md [hunk 5]
```diff
@@ -1,225 +1,364 @@
...
+```
 
-Then, load in the historical data of the company by using the <a href="/terminal/reference/stocks/load/" target="_blank" rel="noreferrer noopener">load</a>. We also choose a different starting period by using `-s` as shown in the documentation:
+### Search
+
+The `search` function provides a way to find stocks by name, region, sector, industry and exchange location. The results can be easily exported as a CSV, JSON, or XLSX file.
+
+
+```console
+search -s technology --country india -e india --export csv
```
Comment (JerBouma) [inline website/content/terminal/guides/intros/stocks/index.md:329]: Let's place this in a separate code block so people can copy.

**Review by JerBouma — CHANGES_REQUESTED** [review]
I am missing some information regarding using a sub-menu. I'd introduce the `fa` menu which was previously done again. You just want to show that this integration with these menus. Next to `fa` you could also do `ta` or `dd` linking to the guides as well if people wish to know more.

File: website/content/terminal/guides/intros/stocks/index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (deeleeramone) [reply website/content/terminal/guides/intros/stocks/index.md:158]: This PR fixes that so it does work: https://github.com/OpenBB-finance/OpenBBTerminal/pull/3655

File: website/content/terminal/guides/intros/stocks/index.md [hunk 6]
```diff
@@ -1,225 +1,364 @@
...
+
+                                          SPY Performance                                           
+┏━━━━━━━━┳━━━━━━━━┳━━━━━━━━━┳━━━━━━━━━┳━━━━━━━━━━┳━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━┓
+┃ 1 Day  ┃ 1 Week ┃ 1 Month ┃ 1 Year  ┃ YTD      ┃ Volatility (1Y) ┃ Volume (10D avg) ┃ Last Price ┃
+┡━━━━━━━━╇━━━━━━━━╇━━━━━━━━━╇━━━━━━━━━╇━━━━━━━━━━╇━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━┩
+│ 2.97 % │ 1.95 % │ 5.56 %  │ -9.16 % │ -13.67 % │ 24.19 %         │ 67.92 M          │ 407.68     │
+└────────┴────────┴─────────┴─────────┴──────────┴─────────────────┴──────────────────┴────────────┘
+```
+
+The load function has many optional arguments. Attach `-h` or `--help` to any function to print the list of specific arguments and their syntax.
```
[the remark this answers is not in the document]
Comment (deeleeramone) [reply website/content/terminal/guides/intros/stocks/index.md:85]: This was already included, but have moved it to the top of the example for `load`.

File: website/content/terminal/guides/intros/stocks/index.md [hunk 7]
```diff
@@ -1,225 +1,364 @@
...
+!![candle](https://user-images.githubusercontent.com/85772166/205148707-f1f2172f-8dee-4f31-b020-4f8043ccccd7.png "candle")
+
+The look and feel of the `candle` chart can be altered with [custom style sheets](https://matplotlib.org/stable/tutorials/introductory/customizing.html#the-default-matplotlibrc-file) placed in `OpenBBUserData/styles/user`.
+
+### News
+
+Get ticker-related news headlines by entering `news` after loading a ticker.
+
+```console
+news --source NewsApi
```
[the remark this answers is not in the document]
Comment (deeleeramone) [reply website/content/terminal/guides/intros/stocks/index.md:205]: done

File: website/content/terminal/guides/intros/stocks/index.md [hunk 8]
```diff
@@ -1,225 +1,364 @@
...
+└──────────┴───────────┴───────────┴──────────┘
 ```
 
-<p align="center">Companies found with the term 'microsoft'</p>
+### Quote
+
+`quote` displays a table with the current price (during market hours) and some general statistics.
+
+```console
+quote
```
[the remark this answers is not in the document]
Comment (deeleeramone) [reply website/content/terminal/guides/intros/stocks/index.md:272]: Done

File: website/content/terminal/guides/intros/stocks/index.md (hunk 5 above)
Comment (deeleeramone) [reply website/content/terminal/guides/intros/stocks/index.md:329]: done

File: website/content/terminal/guides/intros/stocks/index.md (hunk 4 above)
Comment (deeleeramone) [reply website/content/terminal/guides/intros/stocks/index.md:307]: done

File: website/content/terminal/guides/intros/stocks/index.md [hunk 9]
```diff
@@ -1,225 +1,364 @@
...
-show analyst recommendations (<a href="/terminal/reference/stocks/dd/pt/" target="_blank" rel="noreferrer noopener">pt</a>), evaluate an entire sector or industry (<a href="/terminal/reference#sia" target="_blank" rel="noreferrer noopener">sia</a>), and show the historical prices, correlations and sentiment between similar companies (<a href="/terminal/reference/stocks/ca/hcorr/" target="_blank" rel="noreferrer noopener">correlation</a>).
+The Stocks menu is the high-level menu for the public equities asset class. It is divided into two sections:
+
+- Functions for searching, loading, and displaying candles, quotes, and news.
+- Sub-menus
```
[the remark this answers is not in the document]
Comment (deeleeramone) [reply website/content/terminal/guides/intros/stocks/index.md]: Done

**Review by JerBouma — APPROVED** [review]

**Pull request merged**